src/internal/_stepCat.js

```diff
@@ -20,7 +22,7 @@ module.exports = (function() {
   var _stepCatObject = {
     '@@transducer/init': Object,
     '@@transducer/step': function(result, input) {
-      return merge(
+      return _objectAssign(
```

Comment:
Oh, of course.  Interestingly, way back in Ramda's prehistory, the predecessor to `merge` was built on top of an internal version which mutated the first argument, to which we simply supplied `{}` in the public API.  Ah well, _plus ça change..._.